### DISCUSSION TITLE
add new colorscheme

### DISCUSSION THREAD

**Discussion by JacobOtte** [jdhao/nvim-config#336] [Q&A]
pretty new to vim and i used this config to get into nvim. I wondered how i can add new colorschemes?
Thank you for your help.

**Comment by jdhao** [comment]
A list of colorschemes used is [here](https://github.com/jdhao/nvim-config/blob/81f64ed32eb229ffe22cec030ab0126b981ab9aa/lua/plugin_specs.lua#L140), and one of those is loaded [here](https://github.com/jdhao/nvim-config/blob/main/lua/colorschemes.lua)
